docs/02-get-started/01-quick-install.md

```diff
@@ -4,7 +4,20 @@ title: Quick install
 
 To get started with Kyma, let's quickly install it first.
 
-## Install Kyma
+The Kyma project is currently in the transition phase from classic to modular Kyma. You can either install classic Kyma with its components, or available modules. To see the list of Kyma modules, go to [Overview](/docs/01-overview/README.md).
+
+> **CAUTION:** Components transformed into modules aren't installed as part of preconfigured classic Kyma.
+
+## Install a Kyma module
+
+To install a module, deploy its module manager and apply the module configuration. Run:
```

Comment:
These generic commands do not help. Consider moving instructions from the second file or at least provide a link to the second file here (my preference is to merge these 2 files).